frontend/catalyst/api_extensions/quantum_operators.py

```diff
@@ -124,7 +124,7 @@ def circuit():
             return qml.expval(qml.PauliZ(0))
 
     >>> circuit()
-    -1.0
+    Array(-1., dtype=float64)
```

Comment:
Something to note: under #895, qjit functions that return 'floats' now always return `jax.Array`. I _think_ this is preferred but just wanted to double check.

Reply:
I think this is correct.